server/server.go

```diff
@@ -53,7 +55,9 @@ type Server struct {
 	startedTs int64
 	secret    string
 
-	// boot specifies that the server
+	mysqlbinlogDir string
+
+	// boot specifies that whether the server boot correctly
```

Comment:
What does it mean? This comment is too vague.

Reply:
This comment is for boot, https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/server/server.go?L111.
We need stop the embed postgres if some errors occur after start embed postgres